circle.yml

```diff
@@ -15,3 +15,8 @@ test:
         files:
           - client/**/Makefile
           - server/**/Makefile
+    - case $CIRCLE_NODE_INDEX in 0) NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results.xml npm run test-client -- --reporter=mocha-junit-reporter ;; 1) NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results.xml npm run test-server -- --reporter=mocha-junit-reporter ;; esac:
```

Comment:
we could pass the CIRCLE_NODE_INDEX to the test invocation and then have the runner pick every nth test suite to run?